src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -2358,6 +2358,69 @@ var projectUniqueNamesForBuildIntToUpdate
                         }
                     }
 
+#if PACKAGE_CONFIG_LOCK
```

Comment:
Why do you need this if/def? where is this defined?

Reply:
I originally thought to merge into dev, and I didn't want dev builds to use these changes until the feature is ready. I was setting it on my machine only. But since I created a feature branch, it's no longer necessary.